src/main/java/uk/gov/hmcts/reform/profilesync/client/UserProfileClient.java

```diff
@@ -20,7 +20,8 @@
 @FeignClient(name = "userProfileClient", url = "${userprofile.api.url}")
 public interface UserProfileClient {
 
-    @PutMapping(value = "/v1/userprofile/{userId}", consumes = {APPLICATION_JSON_VALUE}, produces = {APPLICATION_JSON_VALUE})
+    @PutMapping(value = "/v1/userprofile/{userId}", consumes = {APPLICATION_JSON_VALUE}, produces =
+            {APPLICATION_JSON_VALUE})
     @Headers({"authorization: {authorization}", "serviceauthorization: {serviceauthorization}"})
```

Comment:
can't see any difference..can we plz revert this in no difference. as it adds unchanged file in change list.